tests/phpunit/api/v3/JobTest.php

```diff
@@ -832,16 +840,16 @@ public function hookMostRecentDonor(&$blocksDAO, $mainId, $otherId, $migrationIn
                 // We also want to be more ruthless about removing matching addresses.
                 unset($mainBlock['location_type_id']);
                 if (CRM_Dedupe_Merger::locationIsSame($block, $mainBlock)
-                  && (!isset($blocksDAO[$blockType]['update']) || !isset($blocksDAO[$blockType]['update'][$mainBlock['id']]))
-                  && (!isset($blocksDAO[$blockType]['delete']) || !isset($blocksDAO[$blockType]['delete'][$mainBlock['id']]))
+                  && (!isset($blocksDAO[$blockType]['update'][$mainBlock['id']]))
```

Comment:
@eileenmcnaughton are we sure that ['update'] and ['delete'] keys will exist?

Reply:
@seamuslee001 it's a test so it will fail if there is an e-notice (it was a clean up my IDE recommended which I didn't analyse carefully as I knew the tests would fail if it a problem & they didn't)

Reply:
that is probably as good as anything maybe it was just something legacy maybe